src/plugins/data/common/es_query/kuery/ast/kuery.peg

```diff
@@ -253,6 +253,7 @@ UnquotedLiteral
 UnquotedCharacter
   = EscapedWhitespace
   / EscapedSpecialCharacter
+  / EscapedUnicodeSequence
```

Comment:
We need to add this to the `QuotedCharacter` rule as well so users can search for something like `"bar\u00A0and baz"`.

Reply:
Thank you, added 👍